Assets/Fungus/Scripts/Commands/SayLocalized.cs

```diff
@@ -0,0 +1,210 @@
+// This code is part of the Fungus library (http://fungusgames.com) maintained by Chris Gregan (http://twitter.com/gofungus).
+// It is released for free under the MIT open source license (https://github.com/snozbot/fungus/blob/master/LICENSE)
+
+// Snippet added by ducksonthewater, 2019-05-29 - www.ducks-on-the-water.com
+
+using UnityEngine;
+
+namespace Fungus
+{
+    /// <summary>
+    /// Writes text in a dialog box.
+    /// </summary>
+    [CommandInfo("Narrative",
+                 "Say Localized",
+                 "Writes text in a dialog box.")]
+    [AddComponentMenu("")]
+    public class SayLocalized : Command, ILocalizable
```

Comment:
This command is a modified copy of the Say command with a few modifications. Given the large amount of overlap between the two commands, it would be better to inherit from the Say command and override the parts that are different. This would avoid the need to keep 2 versions of the Say command in sync as bug fixes and improvements are made.

Reply:
Hi Chris, I reverted my Say-commands within GitHub.
Will have a look on how to improve it.